frontend/src/main-components/InsideFolderContent.js

```diff
@@ -1,22 +1,66 @@
 import React from "react";
 import styled from "@emotion/styled";
+import Flashcard from "../flashcards/Flashcard.js";
+import NewCard from "../flashcards/NewCard.js";
 
+const DebugMessage = (message, isOn) => {
+  if (isOn) {
+    console.log(message);
+  }
+};
+
+const debuggerMessageOn = false;
 
 const InsideFolderContent = (props) => {
   const Container = styled.div`
-    flex: 9;
+    flex: 9 1 auto;
     display: flex;
     border: 1rem solid white;
     border-radius: 1rem;
     background-color: white;
     margin: 1%;
+    flex-direction: column;
   `;
-  
 
+  let flashcards;
+  try {
+    fetch(`/usercards?folderKey=${folderKey}`, { method: "GET" })
+      .then((result) => result.json())
+      .then((data) => {
+        flashcards = data.map((flashcard) => (
+          <Flashcard
+            key={flashcard.cardKey}
+            image={flashcard.blobKey}
+            text={flashcard.rawText}
+            translation={flashcard.textTranslated}
+            labels={flashcard.labels}
+          />
+        ));
+      });
+  } catch (err) {
```

Comment:
I don't think this will ever catch anything -- promises don't throw. Instead, you have to use the [Promise.prototype.catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch) method.

Reply:
fixed